File: plusMinus/plusMinus.cpp
#include <iostream>
#include <vector>

void plusMinus(std::vector<int> arr)
{
	float pos = 0;
	float neg = 0;
	float zero = 0;
	int n = arr.size();

	for (int i = 0; i < n; i++)
	{
		if (arr[i] == 0)
		{
			zero++;
		}
		if (arr[i] < 0)
		{
			neg++;
		}
		if (arr[i] > 0)
		{
			pos++;
		}
	}
	std::cout << pos / n << std::endl;
	std::cout << neg / n << std::endl;
	std::cout << zero / n << std::endl;

}	

int main()
{
	std::vector<int> arr {-4, 3, -9, 0, 4, 1};

	plusMinus(arr);
	

	return 0;

}	

File: plusMinus/plusMinus.py
arr = [-4, 3, -9, 0, 4, 1]


def plusMinus(arr):
    pos = 0
    neg = 0
    zero = 0
    n = len(arr)

    for x in range(0, len(arr)):
        if arr[x] > 0:
            pos = pos + 1
        if arr[x] < 0:
            neg = neg + 1
        if arr[x] == 0:
            zero = zero + 1

    print(pos / n)
    print(neg / n)
    print(zero/ n)

plusMinus(arr)
